script.js

```diff
@@ -0,0 +1,48 @@
+//Part 1
+var productItems=products.items;
+var count=0;
+productItems.forEach(function(item){
+	if(item.kind == "shopping#product")
+		count++;
+});
+console.log(count);
+
+//Part 2
+var itemList = products["items"];
```

Comment:
I'm just curious, there's no right or wrong way to do it.

But why did you use forEach?